Cargo.toml

```diff
@@ -18,22 +17,20 @@ members = [
 
 [workspace.package]
 edition = "2021"
-version = "0.0.1"
-license = "MIT OR Apache-2.0"
+rust-version = "1.75"
```

Comment:
In general we include a comment with the `rust-version` to remind people to also update it in `ci.yml`
Here's what the one out of Peniko looks like: 
```markdown
# If updating, ensure that it is also updated in .github/workflows/ci.yml, and
# update the documented reason
```

Right now, the `ci.yml` asks for 1.76 instead as well, so it might be worth bringing it down to 1.75 there if it is meant to build against 1.75.

Reply:
There is no MSRV job in the Vello CI and Vello doesn't have a MSRV policy yet. I kept this really simple on purpose so that it's specified for Cargo instead of undefined, because the release is imminent. I tested it locally to work with 1.75. Even if it wouldn't actually work with 1.75 (let's say a dep bumped their MSRV just now) then the experience wouldn't differ from it being undefined, as in both cases the compilation would fail.

A more through pass on the documentation and CI would come with [RFC#5](https://github.com/linebender/rfcs/pull/5), which won't be ready in time for the planned release.

PS. Can't bring CI down to 1.75 because bevy requires 1.76. However that doesn't affect our to-be-published crates.